exporter/opentelemetry-exporter-jaeger/src/opentelemetry/exporter/jaeger/__init__.py

```diff
@@ -299,6 +309,13 @@ def _convert_int_to_i64(val):
     return val
 
 
+def _otlp_to_jaeger_span_kind(kind: SpanKind) -> str:
+    jaeger_kind = OTLP_JAEGER_SPAN_KIND.get(kind, "")
+    if not jaeger_kind:
```

Comment:
Would it be better to explicitly check for `INTERNAL` instead?

Reply:
The thinking was that some users or distributions might have custom values for the kind attribute so this would take care of all unknown cases including `INTERNAL` which is unknown as far as Jaeger/OpenTracing is concerned but looks like the spec and API do not support that (even if it is not enforced). So I'm replacing it with a map and adding a test case to ensure all possible values are mapped.